src/components/button/bl-button.css

```diff
@@ -31,7 +31,10 @@
   width: 100%;
   height: var(--height);
   border: solid 1px var(--border-color);
-  border-radius: 6px;
+  border-top-left-radius: var(--bl-border-radius-m);
+  border-top-right-radius: var(--bl-border-radius-m);
+  border-bottom-right-radius: var(--bl-border-radius-m);
+  border-bottom-left-radius: var(--bl-border-radius-m);
```

Comment:
I had considered the possibility of overriding for each border-radius value but it wouldn't be possible since they have the same variable, you're right